posthog/api/dashboards/dashboard.py

```diff
@@ -288,6 +290,9 @@ def update(self, instance: Dashboard, validated_data: dict, *args: Any, **kwargs
         for tile_data in tiles:
             self._update_tiles(instance, tile_data, user)
 
+        instance.filters = initial_data.get("filters", instance.filters)
+        instance.save(update_fields=["filters"])
```

Comment:
This seems wasteful, because it means this method runs `UPDATE posthog_dashboard ...` twice – can we merge this operation with the existing `super().update(instance, validated_data)`?